auth_brute_force/readme/ROADMAP.md

```diff
@@ -0,0 +1,12 @@
+* Remove 🐒 patch for https://github.com/odoo/odoo/issues/24183 in v12.
```

Comment:
Do you think if this roadmap section is still valid ?or obsolete ?

Reply:
done